libheif/CMakeLists.txt

```diff
@@ -89,3 +89,20 @@ if(X265_FOUND)
     target_link_libraries(${PROJECT_NAME} ${X265_LIBRARIES} ${X265_LINKDIR})
 endif()
 
+# Install commands
+# --------------------------
+write_basic_package_version_file(heifConfigVersion.cmake COMPATIBILITY ExactVersion)
```

Comment:
I would prefer `${PROJECT_NAME}-config-version.cmake`

Reply:
I just would like to know if it is just a "preference" or is there some real limitation with the current naming. But I will change it anyways, since I do not have a strong opinion about this point. I am just asking to learn :wink:

Reply:
It is more a preference using the dashes than the camelcase.

Reply:
Alright, I just wanted to be sure I was not missing something. I already pushed the changes 😉